client/reader/search-stream/style.scss

```diff
@@ -137,13 +137,9 @@
 	}
 }
 
-// Custom breakpoints
-$reader-post-card-breakpoint-large: "( min-width: 961px ) and ( max-width: 1040px )";
-$reader-post-card-breakpoint-medium: "( min-width: 661px ) and ( max-width: 940px )";
-$reader-post-card-breakpoint-small: "( max-width: 550px )";
-
 // Post recommendations in Search
-.is-reader-page .search-stream .reader__content {
+.is-reader-page .search-stream .reader__content,
+.is-reader-page .search-stream.search-stream__with-sites .reader__content {
```

Comment:
Had to be more specific here because even though both new and old Search have the `is-reader-page`, `search-stream`, and `reader__content` classes, its properties are only being applied to the current Search. Not sure why?